resources.whatwg.org/build/deploy.sh

```diff
@@ -111,6 +112,32 @@ run_post_build_step "$WEB_ROOT"
 echo "Living standard output to $WEB_ROOT"
 echo ""
 
+header "Starting review drafts..."
+echo "Note: review drafts must be added or changed in a single commit on master"
+for f in "$REVIEW_DRAFTS_DIR"/*.bs; do
+    [ -e "$f" ] || continue # http://mywiki.wooledge.org/BashPitfalls#line-80
+    if [[ "$TRAVIS_PULL_REQUEST" == "true" ]]; then
+        CHANGED_FILES=$(git diff --name-only master..HEAD)
+    else
+        CHANGED_FILES=$(git diff --name-only HEAD~1)
+    fi
+    for change in $CHANGED_FILES; do # Omit quotes around variable to split on whitespace
```

Comment:
FWIW we should probably stick to uppercase everywhere, I was just copying and pasting from places that did lowercase.